src/fontra/client/core/glyph-controller.js

```diff
@@ -648,6 +652,16 @@ export class StaticGlyphController {
       selectionRects.push(component.bounds);
     }
 
+    for (const anchorIndex of anchorIndices) {
+      const anchor = this.instance.anchors[anchorIndex];
+      selectionRects.push({
```

Comment:
You could use `centeredRect()` with width and height set to zero.

Reply:
Thanks. Done.